### PR TITLE
feat/ELE-30: Infinite scroll

### PR DESCRIPTION
**Pull request by olehhapuk**
## Features

- feat: added infinite scroll for chat
- feat: removed query client persist

# Fixes

- fix: fixed dashboard page not working
- fix: fixed create hub dialog crashing web page on close

## Preview

<details>

https://github.com/user-attachments/assets/ef0c228f-5a62-4ff2-8826-dbd76582d5a2

</details>

### CODE REVIEW

**Review by Koriagami — APPROVED** [review]
@olehhapuk da best!

File: src/app/components/app.tsx [hunk 1]
```diff
@@ -1,5 +1,10 @@
+import { Suspense } from 'react';
 import AppProvider from './app-provider';
 
 export default function App() {
-  return <AppProvider />;
+  return (
+    <Suspense>
```
Comment (chaosua-del) [inline src/app/components/app.tsx:6]: Is it good practice to wrap whole application in suspense?

**Review by chaosua-del — APPROVED** [review]

File: src/app/components/app.tsx (hunk 1 above)
Comment (chaosua-del) [reply src/app/components/app.tsx:6]: is yes - just resolve

Comment (olehhapuk) [reply src/app/components/app.tsx:6]: It's okay

**Pull request merged**